CWE/decisions/claim_greater.txt

```diff
@@ -173,7 +177,7 @@ political_decisions = {
 				ruling_party_ideology = populist
 				ruling_party_ideology = nationalist
 				jingoism = 24
-NOT = { government = democracy government = hms_government }
+				NOT = { government = democracy government = democracy1 government = hms_government government = hms_government1 }
```

Comment:
Should be NOT = { OR = { government = democracy government = democracy1 government = hms_government government = hms_government1 } }

Reply:
Thanks!